db/seeds.rb

```diff
@@ -6,27 +6,70 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
-futurama = Video.create(
+comedy = Category.create(name: "Comedy")
+drama = Category.create(name: "Drama")
+
+Video.create(
```

Comment:
You should use `create!` in `seeds.rb` instead of `create` - it will raise an exception in case the object you want to create is invalid. Now it will silently ignore so you won't know if the object is created or not.